docs/apache-airflow-providers-opentelemetry/connections/opentelemetry.rst

```diff
@@ -0,0 +1,40 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection:opentelemetry:
+
+OpenTelemetry Connection
+=============================
+
+The OpenTelemetry Connection type enables integrations with the OpenTelemetry endpoint.
```

Comment:
So, the purpose of metrics in Otel hook is something similar to the datadog hook, where the hook is designed to be used in case user wants to emit their own 'custom metrics' during their dag runs, regardless of the metrics being emitted by airflow.